jest.config.js

```diff
@@ -2,8 +2,18 @@
 // Licensed under the MIT license.
 
 module.exports = {
-  verbose: true,
-  setupFiles: ['./src/setupTests.js'],
-  moduleDirectories: ['node_modules', 'src'],
-  setupFilesAfterEnv: ['./rtl.setup.js'],
+  testMatch: ['**/src/**/*.spec.js'],
+  moduleNameMapper: {
+    '\\.(css|scss|less)$': 'identity-obj-proxy',
+  },
+  transform: {
+    '^.+\\.js?$': 'babel-jest',
+  },
+  testEnvironment: 'jest-environment-jsdom',
+  setupFiles: ['jest-canvas-mock'],
+  setupFilesAfterEnv: ['<rootDir>/jestSetup.js'],
+  collectCoverage: true,
+  coverageDirectory: '<rootDir>/coverage/',
+  coveragePathIgnorePatterns: ['node_modules', 'tests'],
```

Comment:
Not sure what directories are covered yb default, but we may need to ignore the `cypress` and `scripts` folders too

Reply:
Right, i changed this by collectCoverageFrom: ['src/**/*.js'],